src/model.cpp

```diff
@@ -195,4 +198,22 @@ Status Model::reloadVersions(std::shared_ptr<model_versions_t> versionsToReload,
     return result;
 }
 
+Status Model::cleanupModelTmpFiles(const ModelConfig& config) {
+    auto lfstatus = StatusCode::OK;
+
+    if (config.getLocalPath().compare(config.getBasePath())) {
```

Comment:
I think this check is not self-explanatory & require a lot of passing pointers. What would be better I think:
bool ModelConfig::isCloudStored()
which would do this check there.